index.html

```diff
@@ -47,8 +47,11 @@ <h2>Projects</h2>
                 <h3>Instagram</h3>
                 <p>You can post pictures online.</p>
                 <ul>
-                    <li>Perl</li>
-                    <li>Assembler</li>
+                    <li><img class="logo" src="img/html5.svg" alt="html5 logo"></li>
+                    <li><img class="logo" src="img/css3.svg" alt="css3 logo"></li>
+                    <li><img class="logo" src="img/javascript.svg" alt="javascript logo"></li>
+                    <li><img class="logo" src="img/php.png" alt="php logo"></li>
+                    <li><img class="logo" src="img/mysql.svg" alt="mysql logo"></li>
```

Comment:
I wrote it in another PR of yours, this `logo` at the end in the alt text is not necessary. The alt is read out loud by screen readers and so it would read `html5 logo` although you actually just want to emphasis that you used html *not* that it is a logo.
```suggestion
                    <li><img class="logo" src="img/html5.svg" alt="html5"></li>
                    <li><img class="logo" src="img/css3.svg" alt="css3"></li>
                    <li><img class="logo" src="img/javascript.svg" alt="javascript"></li>
                    <li><img class="logo" src="img/php.png" alt="php"></li>
                    <li><img class="logo" src="img/mysql.svg" alt="mysql"></li>
```